docs/cas-server-documentation/logging/Logging-SysLog.md

```diff
@@ -29,6 +29,10 @@ messages needs to be routed over to this instance:
 
 ```
 
+<div class="alert alert-warning">
+  <strong>Pay Attention</strong><br /> [Mapped Diagnostic Context](Logging-MDC.html) (`MDC`) may contain the password. Setting `includeMDC=true` sends clear password as variable to syslog.
```

Comment:
This is rendered as HTML, not markdown. You should be using HTML tags for links and for `<code>`

Reply:
I've made another commit to change the link. Sorry for the multiple mistakes. This will at least make me learn.